README.md

```diff
@@ -1,126 +1,174 @@
+  print(`PlayerData replica received! Received player money: ${replica.Data.Money}`);
 
-const replica = ReplicaService.NewReplica({
-	ClassToken: token,
-	Data: {
-		Cans: 5,
-	},
-	WriteLib: writeLib,
+  replica.ListenToChange(["Money"], (newValue) => {
+    print(`Money changed: ${newValue}`);
+  });
 });
 
-replica.Write(["RestockAll"]);
-
+ReplicaController.RequestData(); // This function should only be called once in the entire codebase! Read the documentation for more information.
 ```
 
-# [ReplicaService Basic Usage](https://madstudioroblox.github.io/ReplicaService/tutorial/basic_usage/) translated to TS
+## ✨ Recommendations
 
-A .d.ts file
-```ts
-declare global {
-	interface Replicas {
-		TestReplica: {
-			Data: { Value: number };
-			Tags: {};
-			WriteLib: {};
-		};
-	}
-}
-```
+- Make your `Replica.Data` simple and small without too many keys inside another keys.
 
-Server
-```ts
-import { ReplicaService } from "@rbxts/replicaservice";
+## ⛔ Limitations
 
-const test_replica = ReplicaService.NewReplica({
-	ClassToken: ReplicaService.NewClassToken("TestReplica"),
-	Data: { Value: 0 },
-	Replication: "All",
-})
+- Paths (`StringPath` and `ArrayPath`) can only access **21 keys** of an object (this was added as a fix to the issue "Type instantiation is excessively deep and possibly infinite").
 
-while (wait(1)) {
-	test_replica.SetValue(["Value"], test_replica.Data.Value + 1);
-}
-```
+## ❔ Frequently Asked Questions
 
-Client
-```ts
-import { ReplicaController } from "@rbxts/replicaservice";
-ReplicaController.ReplicaOfClassCreated("TestReplica", (replica) => {
-	print("TestReplica received! Value:", replica.Data.Value);
+1. **My editor features (autocomplete, others) are laggy, what can I do?** Reopen your code editor (or if you're using Visual Studio Code, restart the TypeScript server), if it's still laggy, contact any of the collaborators in the `roblox-ts` server.
+2. **I can't access a key in my object that is inside 35 keys!** Read the [limitations](#-limitations) and [recommendations](#-recommendations).
 
-	replica.ListenToChange(["Value"], (new_value) => {
-		print("Value changed:", new_value);
-	});
-});
+## 📜 Documentation
+
+Visit the following website to go to the official ReplicaService documentation: https://madstudioroblox.github.io/ReplicaService/
+
+## ⚠️ Support
 
-ReplicaController.RequestData(); // This function should only be called once
-//   in the entire codebase! Read the documentation for more info.
+If you are having issues with typings, join `roblox-ts` Discord server and mention any of the collaborators ([Mixu_78](https://discord.com/users/255257883250393091), or [Sandy Stone](https://discord.com/users/1018447375079063573)) with your issue and we'll try to help the maximum we can.
 
-```
\ No newline at end of file
+If you are having issues with ReplicaService and not the typings, [file an issue in the GitHub repository](https://github.com/MadStudioRoblox/ReplicaService/issues).
```

Comment:
Duplicating the ReplicaService description is a bit unnecessary, some stuff like default.project.json could also be removed from the examples here